common/entrypoint_mautic_worker.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # wait until Mautic is installed
-until php -r 'file_exists("/var/www/html/config/local.php") ? include("/var/www/html/config/local.php") : exit(1); exit(isset($parameters["site_url"]) ? 0 : 1);'; do
+until php -r "include('$MAUTIC_VOLUME_CONFIG/local.php'); exit(isset(\$parameters['site_url']) ? 0 : 1);"; do
```

Comment:
At this point the local.php has to exist or the script couldn't be running so we removed that check